Tests/SwiftGenKitTests/TestsHelper.swift

```diff
@@ -187,11 +187,34 @@ class Fixtures {
     let path = self.path(for: name, subDirectory: "StencilContexts/\(sub.rawValue)")
 
     guard let data: String = try? path.read(),
-      let yaml = try? Yams.load(yaml: data, .basic),
+      let yaml = try? Yams.load(yaml: data),
       let result = yaml as? [String: Any] else {
         fatalError("Unable to load fixture content")
     }
 
     return result
   }
 }
+
+// MARK: - Temporary fix until Yams fixes string dumping
+
+enum YamsFix {
+  static func represent(_ any: Any) throws -> Node {
+    switch any {
+    case let string as String:
+      // if the tag after resolving is not "str", then this string is ambiguous
+      let tag = Resolver.`default`.resolveTag(of: Node(string))
+      let style: Node.Scalar.Style = (tag != .str) ? .doubleQuoted : .any
+      return Node(string, .implicit, style)
```

Comment:
I wonder if this test is worth it (as oppose to dump all strings as double quoted, regardless if they're ambiguous or not)?

Reply:
Well it addresses your comment from the original code:
> // maybe only if `["true", "false"].contains(string) ? Or for all strings?

Wether it should be done or not, dunno. YAML doesn't quote strings by default, so I tried to emulate that behaviour.

Reply:
Tbh when I first read the did on GitHub the first diff shown is the context for colors, and I wondered why / found it odd that `"66"` was quoted but `ff` wasn't 😜

```diff
  - colors:
    - alpha: ff
 -    blue: 66
 -    green: 96
+     blue: "66"
+     green: "96"
      name: ArticleBody
-     red: 33
+     red: "33"
```
(then saw your implementation later in the diff and got why)
Maybe quoting everything would be more consistent and eye-pleasing? 🧐😛